beacon-chain/operations/attestations/kv/aggregated_test.go

```diff
@@ -256,7 +256,7 @@ func TestKV_Aggregated_AggregatedAttestations(t *testing.T) {
 	sort.Slice(returned, func(i, j int) bool {
 		return returned[i].Data.Slot < returned[j].Data.Slot
 	})
-	assert.DeepEqual(t, atts, returned)
+	assert.DeepSSZEqual(t, atts, returned)
```

Comment:
We can no longer check protobuf arrays with reflect, so every check for protobuf arrays now goes through sszutil. Is this fine?  I don't think I've changed any runtime code.

Reply:
can you explain why reflect can't catch this ? The underlying data structure should be the same here.